[2024-11-25T16:22:13.632+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-25T16:22:13.734+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelamento.save_data scheduled__2024-11-25T15:30:00+00:00 [queued]>
[2024-11-25T16:22:13.826+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelamento.save_data scheduled__2024-11-25T15:30:00+00:00 [queued]>
[2024-11-25T16:22:13.826+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 4
[2024-11-25T16:22:13.936+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): save_data> on 2024-11-25 15:30:00+00:00
[2024-11-25T16:22:13.974+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=290) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-25T16:22:13.997+0000] {standard_task_runner.py:72} INFO - Started process 345 to run task
[2024-11-25T16:22:14.000+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'get_inaturalist_data_atropelamento', 'save_data', 'scheduled__2024-11-25T15:30:00+00:00', '--job-id', '1464', '--raw', '--subdir', 'DAGS_FOLDER/inaturalist_dag.py', '--cfg-path', '/tmp/tmptus8_ov3']
[2024-11-25T16:22:14.005+0000] {standard_task_runner.py:105} INFO - Job 1464: Subtask save_data
[2024-11-25T16:22:14.275+0000] {task_command.py:467} INFO - Running <TaskInstance: get_inaturalist_data_atropelamento.save_data scheduled__2024-11-25T15:30:00+00:00 [running]> on host 07f92f2579af
[2024-11-25T16:22:14.759+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='get_inaturalist_data_atropelamento' AIRFLOW_CTX_TASK_ID='save_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-25T15:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-25T15:30:00+00:00'
[2024-11-25T16:22:14.766+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-25T16:22:14.769+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-25T16:22:14.770+0000] {logging_mixin.py:190} INFO - Current task name:save_data state:running start_date:2024-11-25 16:22:13.737027+00:00
[2024-11-25T16:22:14.770+0000] {logging_mixin.py:190} INFO - Dag name:get_inaturalist_data_atropelamento and current dag run status:running
[2024-11-25T16:22:14.771+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-25T16:22:14.776+0000] {inaturalist_dag.py:196} INFO - No new observations to store.
[2024-11-25T16:22:14.777+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-25T16:22:14.820+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-25T16:22:14.821+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=get_inaturalist_data_atropelamento, task_id=save_data, run_id=scheduled__2024-11-25T15:30:00+00:00, execution_date=20241125T153000, start_date=20241125T162213, end_date=20241125T162214
[2024-11-25T16:22:14.851+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-25T16:22:14.852+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-25T16:22:14.852+0000] {logging_mixin.py:190} INFO - Dag name:get_inaturalist_data_atropelamento queued_at:2024-11-25 16:09:55.500283+00:00
[2024-11-25T16:22:14.853+0000] {logging_mixin.py:190} INFO - Task hostname:07f92f2579af operator:_PythonDecoratedOperator
[2024-11-25T16:22:14.897+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-25T16:22:14.949+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-25T16:22:14.955+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
